test-cypress/integration/repository/ontop-repository.spec.js

```diff
@@ -137,6 +139,91 @@ describe('Ontop repositories', () => {
         OntopRepositorySteps.getTestConnectionButton().should('not.be.disabled');
     });
 
+    it('should create ontop repository', () => {
+        stubRepoCreationEndpoints();
+        // When I visit the create ontop page
+        OntopRepositorySteps.visitCreate();
+        // The test connection button should be disabled
+        OntopRepositorySteps.getTestConnectionButton().should('be.disabled');
+        // Then I fill the repository ID
+        RepositorySteps.getRepositoryIdField().type(repositoryId);
```

Comment:
Implement methods like `writeRepositoryId(repositoryId)` in the steps file. All low level interactions with the web page must be in the steps.